src/libraries/System.Drawing.Common/pkg/buildTransitive/System.Drawing.Common.targets

```diff
@@ -0,0 +1,7 @@
+<Project InitialTargets="_ErrorForSystemDrawingCommonOnNetCoreApp2x">
+  <Target Name="_ErrorForSystemDrawingCommonOnNetCoreApp2x"
+          Condition="'$(SuppressTfmSupportBuildWarnings)' == ''">
+    <Error Text="System.Drawing.Common doesn't support $(TargetFramework). Consider updating your TargetFramework to netcoreapp3.0 or later."
```

Comment:
Should we suggest `netcoreapp3.1` or later given that 3.0 is out of support and 3.1 is `LTS`?

Reply:
Let's not lie about the supported platforms as technically netcoreapp3.0 is supported by this package. With net7 we will hopefully remove all netcoreapp2.x/netcoreapp3.x assemblies anyway from our packages.

Reply:
Yeah that makes sense in the perspective that this package supports 3.0, I just didn't know if it was a good practice to suggest a tfm that is out of support.

Reply:
> technically netcoreapp3.0 is supported by this package

I don't believe this is correct, but will follow up offline to verify.

Reply:
Note the wording that I used. Technically the package supports netcoreapp3.0 and later, but we recommend to use platforms which are still in support which .NET Core 3.0 itself isn't anymore.

Giving this more thought I agree that it might cause less troubles to change to netcoreapp3.1. But what complicates this whole topic is that we still have TONS of netcoreapp2.x and netcoreapp3.0 assets in the main branch. I considered upgrading all of those and filed https://github.com/dotnet/runtime/issues/46400 some months ago but I'm unsure if we still want to do this in the NET6 timeframe. Same applies to upgrading all assets to net461 for .NET Framework and .NET Standard libs to netstandard2.0.

I would be very happy if we would agree on that this is important to do for 6.0 (LTS) but I'm skeptical that we would get approval from directors.

cc @ericstj @danmoseley

Reply:
cc @terrajobst